src/components/input.js

```diff
@@ -359,6 +358,6 @@ export const TextArea = ({ onChange, ...props }) => {
     as: 'textarea',
     className: 'focus-style',
     style: styles.textarea,
-    onChange: onChange ? (e => onChange(e.target.value)) : undefined
+    onChange: e => onChange(e.target.value)
```

Comment:
nit: React will complain in the console if you try to make a `readOnly` component with an `onChange` handler, which is why it was written this way. I don't think we actually do this anywhere currently, but it's good to be well-behaved for this case anyway.